bmu/Src/sense.c

```diff
@@ -16,7 +16,7 @@
 #include "watchdog.h"
 
 /*#define SENSOR_TASK_PERIOD 50*/
-#define SENSOR_TASK_PERIOD 1000
+#define SENSOR_TASK_PERIOD 1009
```

Comment:
Don't like this non round value

Reply:
See above for why I used non round values. Let me know If there is a different approach